Missing feature flag io-util in tokio

Hi,
thanks for your work on twitchchat. I had a compile error pop up when trying to upgrade to 0.8, and it seems like it's caused by io-util only being enabled in dev-dependencies.

Steps to replicate:
1. Create a new Rust project
2. Add a dependency for twitchchat 0.8
3. Add "use twitchchat;" to main.rs
```rust
error[E0432]: unresolved import `tokio::io::AsyncWriteExt`
   --> [...]/twitchchat-0.8.2/src/lib.rs:120:13
    |
120 |         use tokio::io::AsyncWriteExt as _;
    |             ^^^^^^^^^^^-------------^^^^^
    |             |          |
    |             |          help: a similar name exists in the module: `AsyncWrite`
    |             no `AsyncWriteExt` in `io`
```

Adding io-utils to tokio's features solves the issue. This seems to be a bug in cargo (https://github.com/rust-lang/cargo/issues/1796), where dev-dependency features leak if the dependency is also defined in the regular dependencies. This causes the crate to build fine on its own, but error when used as a dependency.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -25,7 +25,7 @@ futures    = { version = "0.3", optional = true, default-features = false, featu
 log        = { version = "0.4" }
 native-tls = { version = "0.2", optional = true }
 serde      = { version = "1",   optional = true, features = ["derive"] }
-tokio      = { version = "0.2", optional = true, default-features = false, features = ["dns", "stream", "sync", "tcp", "time"] }
+tokio      = { version = "0.2", optional = true, default-features = false, features = ["dns", "io-util", "stream", "sync", "tcp", "time"] }
 tokio-tls  = { version = "0.3", optional = true }
 
 [dev-dependencies]

